client/src/www/model/platform_error.ts

```diff
@@ -14,31 +14,36 @@
 
 import {CustomError} from '@outline/infrastructure/custom_error';
 
+import * as errors from './errors';
+
 /**
  * @fileoverview This file defines types and constants corresponding to the backend Go's
  * `platerrors` package. It will be used to receive native errors from Go, eventually replacing
  * the older NativeError type.
```

Comment:
```suggestion
 * @fileoverview This file defines types and constants corresponding to the backend Go's
 * `platerrors` package. It will be used to receive native errors from Go, eventually being
 * transformed into a strongly-typed `errors.NativeError`.
```